plugins/analysis-nori/src/main/java/org/elasticsearch/plugin/analysis/nori/NoriTokenizerFactory.java

```diff
@@ -80,7 +82,8 @@ public static UserDictionary getUserDictionary(Environment env, Settings setting
      * that the duplicate check feature is supported.
      */
     private static boolean isSupportDuplicateCheck(IndexSettings indexSettings) {
-        return indexSettings.getIndexVersionCreated().onOrAfter(IndexVersions.ES_VERSION_8_13);
+        var idxVersion = indexSettings.getIndexVersionCreated();
+        return idxVersion.onOrAfter(IndexVersions.NORI_DUPLICATES) && idxVersion != UPGRADE_LUCENE_9_9_1;
```

Comment:
Maybe we need a `IndexVersion::not`. !

Reply:
Worth a comment on what's going on here, but this covers what we need

Reply:
Actually, this will mean that it will say the dup check is not supported by indexes created when 8500008 was the highest version, which is wrong.

This is why backporting versions is so tricky...

Reply:
> Actually, this will mean that it will say the dup check is not supported by indexes created when 8500008 was the highest version, which is wrong.

That is true, but does it matter? ;-).  I say this because, this PR is adding two version constants in the same commit - and current/latest will point to the greater of the two. Therefore, there should never be a deployment of ES where Index version is 8500008, unless it is 8.12. Does this make sense ?

Reply:
To restate: the latest serverless deployment is on 8500007. The next serverless deployment will have the 9.9.1 upgrade, and also this PR when merged - so it will have 8500008 and 8500009.

Reply:
Yup, that makes sense. This is all ok then